content/en/docs/concepts/extend-kubernetes/operator.md

```diff
@@ -15,13 +15,13 @@ Kubernetes principles, notably the [control loop](/docs/concepts/architecture/co
 
 ## Motivation
 
-The Operator pattern aims to capture the key aim of a human operator who
+The operator pattern aims to capture the key aim of a human operator who
```

Comment:
(nit)
```suggestion
The _operator pattern_ aims to capture the key aim of a human operator who
```

Reply:
I like your suggestion there, thank you! I will push that change to the branch.